nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -68,7 +68,6 @@ let
     PrivateMounts = true;
     # System Call Filtering
     SystemCallArchitectures = "native";
-    SystemCallFilter = "~@clock @cpu-emulation @debug @keyring @module @mount @obsolete @reboot @resources @setuid @swap";
```

Comment:
Could you abstract over these long repeating strings in the form of a list like `[ "@clock" "@cpu-emulation" ... ]` and then use it like `SystemCallFilter = "~" + lib.concatStringsSep " " (systemCallsList ++ [ "@privileged" ]`? That would reduce redundancy and would make it easy to spot, which systemcalls are denied for a unit besides the base set.

Also could you please write a more specific commit message?

Reply:
Update PR.